Dockerfile

```diff
@@ -1,15 +1,21 @@
 FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
 
-ENV SOLR_VERSION 7.7.2
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE="https://github.com/ArkCase/ark_solr" \
+      MAINTAINER="Armedia LLC"
 
-ENV SOLR_URL https://archive.apache.org/dist/lucene/solr/$SOLR_VERSION/solr-$SOLR_VERSION.tgz
+ENV SOLR_VERSION="7.7.2" \
+    SOLR_USER="solr" \
+    SOLR_GROUP="solr"
 
+ENV SOLR_DOWNLOAD_URL https://archive.apache.org/dist/lucene/solr/${SOLR_VERSION}/solr-${SOLR_VERSION}.tgz
 ENV SOLR_PORT 8983
 
-ADD $SOLR_URL /tmp
+ADD $SOLR_DOWNLOAD_URL /tmp
 
 
-RUN useradd  --system --user-group solr && \
+RUN useradd  --system --user-group $SOLR_USER  && \
```

Comment:
Please explicitly specify the group as `$SOLR_GROUP`, or alternatively do not define the `SOLR_GROUP` environment variable.